src/marqo/tensor_search/tensor_search.py

```diff
@@ -627,6 +631,19 @@ def add_documents(config: Config, index_name: str, docs: List[dict], auto_refres
                     })
 
             # Add chunks_to_append along with doc metadata to total chunks
+
+            elif isinstance(field_content, dict):
+                chunks_to_append, document_is_valid, unsuccessful_doc_to_append, vectorise_time_to_add = \
```

Comment:
Can we have completely new names of objects returned from the function? Name clashes might have unexpected results. In this case, "document_is_valid" in particular, but all should have unique names that gives the reader information about what the objects is (e.g: `vectorise_time_to_add ` -> `combo_vectorise_time_to_add`)

Reply:
Also chunks to append (plural chunks) isn't accurate right? `combo_chunk_to_append` is more accurate, right?

Reply:
updated